### ISSUE TITLE
Document testing

### ISSUE THREAD

**Issue by javigomez** [joomla-projects/com_localise#99]
It would be awesome if you @zanardigit or @puneet0191 could create here: https://github.com/joomla-projects/com_localise/tree/develop/tests/system a readme.md file that explains step by step how to run the tests. Something like we have at http://docs.joomla.org/Running_Automated_Tests_for_the_Joomla_CMS but smaller, just with a few steps to set up.

**Comment by zanardigit** [comment]
Done, PR sent.

**Comment by javigomez** [comment]
Thanks, adding here the part mentioned by @puneet0191  in the pull request: https://github.com/joomla-projects/com_localise/pull/100#issuecomment-45406055

To close this issue we need some one to document how to configure the configuration file. By the way I think we forgot to add the configuration.php.dist example file like we have in joomla cms (https://github.com/redCOMPONENT-COM/redSHOP/blob/develop/tests/system/servers/configdef.php.dist) I don't see we have any in the repository

cc: @zanardigit

**Comment by puneet0191** [comment]
yes, we forgot to add that file..and isn't the configuration file self explainatory? or do we need a doucment to explain it?

**Comment by javigomez** [comment]
yes, the configuration file is self-explanatory, just mention in document were the people can find the configuration file

**Comment by puneet0191** [comment]
okay, will update the document with details of configuration file
